microsoft-edge/progressive-web-apps-chromium/how-to/web-app-manifests.md

```diff
@@ -73,7 +73,8 @@ A PWA can be customized even further by using other manifest members, such as th
 | `categories` | The list of categories the app belongs to.  |
 | `icons` | Array of icon image objects that are used by the OS in different contexts. |
 | `screenshots` | Array of screenshot image objects, also used by the OS in different contexts.  |
-| `lang` | The primary language of the app. |
+| `lang` | The language used by other localized manifest members such as `name` and `short_name`. |
+| `dir` | The text direction used by other localized manifest members such as `name` and `short_name`. |
```

Comment:
```suggestion
| `dir` | The text direction used by localized members of this manifest, such as `name` and `short_name`. |
```
alts:
The text direction used by localized members of this manifest, such as `name` and `short_name`.
The text direction used by other localized manifest members such as `name` and `short_name`.